code/include/playfab_c/PlayFabClientApi_c.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "httpClient/pal.h"
+#include "httpClient/async.h"
+#include "PlayFabClientDataModels_c.h"
+
+/// <summary>
+/// Initialize PlayFab global state
+/// </summary>
+HRESULT PlayFabInitialize(
+    _In_ const char* titleId
+) noexcept;
+
+/// <summary>
+/// Cleanup PlayFab global state. Shouldn't be called while async APIs are still running.
+/// </summary>
+HRESULT PlayFabCleanupAsync(
+    _In_ XAsyncBlock* async
+) noexcept;
+

```

Comment:
consider returning handle so there's no static inside.  similar to lobby here: [https://microsoft.visualstudio.com/Xbox/_git/Xbox.Bumblelion?path=%2Fsrc%2Flobby%2Fpub%2FLobby_c.h&version=GBdevelop&line=257&lineEnd=258&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents](https://microsoft.visualstudio.com/Xbox/_git/Xbox.Bumblelion?path=%2Fsrc%2Flobby%2Fpub%2FLobby_c.h&version=GBdevelop&line=257&lineEnd=258&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents)

Reply:
I like this pattern.  I will try to move to that here as well.

---
In reply to: [590768852](https://github.com/jplafonta/XPlatCppSdk/pull/2#discussion_r590768852) [](ancestors = 590768852)